impeller/renderer/backend/vulkan/gpu_tracer_vk.h

```diff
@@ -81,4 +86,50 @@ class GPUTracerVK {
   bool enabled_ = false;
 };
 
+struct GPUProbe {
```

Comment:
Given this behaves more class-like, I'd just make it a class
(https://google.github.io/styleguide/cppguide.html#Structs_vs._Classes)

Reply:
Good call. It did start out more struct like...